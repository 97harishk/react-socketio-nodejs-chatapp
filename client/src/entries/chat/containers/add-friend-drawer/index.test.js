import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import {
	AddFriendDrawer
} from 'entries/chat/containers';

describe('<AddFriendDrawer />', () => {
	const modelI = {};

	const getWrapper = (model) => {
		return shallow((
			<AddFriendDrawer {...model} />
		));
	};

	it('renders correctly', () => {
		const wrapper = getWrapper(modelI);
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
