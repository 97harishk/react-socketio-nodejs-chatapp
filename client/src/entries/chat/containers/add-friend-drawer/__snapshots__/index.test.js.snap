// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddFriendDrawer /> renders correctly 1`] = `
<div
  className="add-friend-drawer-wrapper"
>
  <div
    className="form-container"
  >
    <InputComponent
      autoComplete="off"
      defaultButton={true}
      margin="0px 0px 15px 0px"
      maxLength={15}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      placeholder="Nickname"
      type="text"
    />
    <ButtonComponent
      primary={true}
      text="Add"
      type="button"
      width="100%"
    />
  </div>
</div>
`;
